pages/activity.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { memo } from 'react';
+import { Abi, getAddress } from 'viem';
+import type { GetStaticProps } from 'next';
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { readdir, readFile } from 'fs/promises';
+
+import Feed from 'components/RiskFeed/Feed';
+import { usePageView } from 'hooks/useAnalytics';
+import { goerli, mainnet, optimism } from 'wagmi/chains';
+import { Contracts } from 'components/RiskFeed/Decode';
+import { basename } from 'path';
+
+type Props = {
+  contracts: Contracts;
+};
+
+const Activity = ({ contracts }: Props) => {
+  usePageView('/activity', 'Activity');
+  const { t } = useTranslation();
+
+  return (
+    <Box my={4} maxWidth={800} mx="auto">
+      <Box>
+        <Typography
+          component="h1"
+          variant="h1"
+          fontSize={24}
+          fontWeight={700}
```

Comment:
this could be `variant="h5"`, without fontSize and fontWeight, but we'd have to update `fontWeight` to 700 in the `theme.ts` (and change the other four occurrences)